src/api/test-helpers/create-multiple-full-applications.ts

```diff
@@ -0,0 +1,21 @@
+import createFullApplication from './create-full-application';
+import { Context } from '../types';
+
+/**
+ * createMultipleFullApplications
+ * Create multiple full applications for unit testing
+ * @param {Object} KeystoneJS context API
+ * @returns {Array<Application>} Applications array
+ */
+export const createMultipleFullApplications = async (context: Context) => {
+  const application1 = await createFullApplication(context);
+  const application2 = await createFullApplication(context);
+  const application3 = await createFullApplication(context);
+  const application4 = await createFullApplication(context);
+  const application5 = await createFullApplication(context);
+  const application6 = await createFullApplication(context);
```

Comment:
New `createMultipleFullApplications` test helper? Something like 


```js
const createMultipleFullApplications = async (count: number) => {
  const arr = new Array(count).fill();

  arr.forEach(async () => {
    await createFullApplication();
  });
};
```

`Promise.all()` would be even better but i'm not sure if that could work right now without refactoring `createFullApplication`.

Reply:
this will be done later, was not working